apps/api/src/admin/participant_manager.py

```diff
@@ -50,9 +50,16 @@ async def get_hackers() -> list[Participant]:
 async def check_in_applicant(uid: str, associate: User) -> None:
     """Check in applicant at IrvineHacks"""
     record: Optional[dict[str, object]] = await mongodb_handler.retrieve_one(
-        Collection.USERS, {"_id": uid, "role": Role.APPLICANT}
+        Collection.USERS, {"_id": uid, "role": {"$exists": True}}
     )
-    if not record or record["status"] not in (Status.ATTENDING, Status.CONFIRMED):
+    if not record:
+        raise ValueError
+
+    status = record.get("status", "")
+    if not status:
+        raise RuntimeError
+
```

Comment:
A `RuntimeError` is usually used in our codebase to indicate an exception out of our control such as a database operation failing. We can use the same `ValueError` here by just omitting these lines and letting the next `not in` condition handle it since `None` is not in the tuple.

Reply:
QQ: Since we have updated all Users to have a status attribute, wouldn't a user not having a status signify an internal server error? If I switch this to raise a `ValueError`, a 404 will be returned which doesn't seem right.

Reply:
There are some user records who actually won't have statuses such as people who logged in but never applied. We could change the parent to status 400 if that would make the error more generic, but I think 404 still kinda makes sense, as in the person the user asked to check is found not be a participant?

Reply:
Ah gotcha, I was missing that part. I'll go ahead and make the change.